hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/volume/TestHddsVolume.java

```diff
@@ -369,6 +369,23 @@ public void testDbStoreClosedOnBadVolumeWithDbVolumes() throws IOException {
     assertEquals(0, DatanodeStoreCache.getInstance().size());
   }
 
+  @Test
+  public void testFailedVolumeSpace() throws IOException {
+
+    // Build failed volume
+    HddsVolume volume = volumeBuilder.failedVolume(true).build();
+
+    // In case of failed volume all stats should return 0.
+    assertEquals(0, volume.getVolumeInfoStats().getUsed());
+    assertEquals(0, volume.getVolumeInfoStats().getAvailable());
+    assertEquals(0, volume.getVolumeInfoStats().getCapacity());
+    assertEquals(0, volume.getVolumeInfoStats().getReserved());
+    assertEquals(0, volume.getVolumeInfoStats().getTotalCapacity());
+
+    // Shutdown the volume.
+    volume.shutdown();
```

Comment:
Cleanup should be performed in `finally` block.

Reply:
Hi @adoroszlai , Thanks for the review. All the comments have been fixed.